### ISSUE TITLE
How to change background color?

### ISSUE THREAD

**Issue by alsoijw** [micheleg/dash-to-dock#313]
How to change background color?

**Comment by micheleg** [comment]
Hi,

there's no option to change the dash background. I'm leaving the styling to the theme authors. However, you can tweak the color manually with a css rule . Have a look at the penultimate line here:  https://github.com/micheleg/dash-to-dock/blob/a82c45213d799f27c68f2c02417b418d9303659e/stylesheet.css

This file is installed in:
```
~/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/

```
You can change the color there and reload the extension.

**Comment by alsoijw** [comment]
How about add GUI setting for this?

**Comment by micheleg** [comment]
It's not in my priority list, as I'm inheriting the color from the theme, but I could add this functionality in the future. But I can't promise anything.

**Comment by SeventhSonOfASeventhSon** [comment]
An option allowing changing of the background colour would be nice.

**Comment by Sancho-Panza** [comment]
Coming soon, see [this PR](https://github.com/micheleg/dash-to-dock/pull/318). ;)

**Comment by micheleg** [comment]
This functionality has now been merged into master [7db860c326567b4d7d17dc3c3fc655543e84e61b].
